pages/blog/announcing-graphql-yoga-2.mdx

```diff
@@ -0,0 +1,278 @@
+import withArticle from '../../ui/blog/article'
+
+export const meta = {
+  title: 'Announcing GraphQL Yoga 2.0!',
+  tags: ['server', 'envelop', 'node', 'graphql'],
+  author: 'guild',
+  date: '2022-03-29',
+  description:
+    'Fully-featured GraphQL Server with focus on easy setup, performance and great developer experience',
+  image: '/blog-assets/angular-cli-meteor-no-eject/cover.png'
+}
+
+export default withArticle({ ...meta })
+
+Today we are incredibly excited to share with you the new GraphQL Yoga! This release was made possible with your contributions, issues, and feedback.
+
+The Guild took over the development of GraphQL Yoga in early 2021, and with the growing community of tools in the GraphQL space, most recently [Envelop](https://envelop.dev/), we were able to rewrite GraphQL Yoga 2.0 from scratch with easy setup, performance, and developer experience at the core.
+
+GraphQL Yoga continues to boast a "convention over configuration" approach and the freedom to use your favorite libraries, from HTTP to schema-building.
+
+No longer are you required to install dozens of dependencies to make features such as Subscriptions, File Uploads, CORS, Error Masking, and more.
```

Comment:
"make features"? Not exactly sure what you wanted to say here, but here are three different possibilities: 

> to make features such as Subscriptions, File Uploads, CORS, Error Masking and more possible

> to get features such as Sub...

> to make features such as Subscriptions, File Uploads, CORS, Error Masking and more easy to build

Reply:
```suggestion
No longer are you required to install dozens of dependencies to get features such as Subscriptions, File Uploads, CORS, Error Masking, and more.
```